web_widget_model_disable_nbrecords_computation/static/src/xml/base.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<template>
+    <t
+        t-name="web_widgetmodel_disable_nbrecords_computation.FieldDomain.content"
+        t-inherit="web.FieldDomain.content"
+        t-inherit-mode="extension"
+    >
+        <xpath expr="//div[hasclass('o_field_domain_panel')]" position="replace">
+            <div t-if="hasModel" class="o_field_domain_panel">
+                <button
+                    t-if="isValid &amp;&amp; nbRecords>=0"
+                    class="btn btn-sm btn-secondary o_domain_show_selection_button"
+                    type="button"
+                >
+                    <i
+                        class="fa fa-arrow-right"
+                        role="img"
+                        aria-label="Domain"
+                        title="Domain"
+                    />
+                    <t t-esc="nbRecords" /> record[s] </button>
+                <span t-elif="isValid &amp;&amp; nbRecords==-1">
+                    Fetch nb records disabled
+                </span>
```

Comment:
I see in the screen captures that the hyperlink is not available on this new text. Any reason for that? Doesn't it makes sense to let the user check the result of the domain even if we don't provide a counter?


Otherwise, I don't expect the user to understand what this message mean and domain editor might be available to anyone. I would go for a more simplistic message.

By simply removing the counter?

```suggestion
            <button t-elif="isValid &amp;&amp; nbRecords==-1" class="btn btn-sm btn-secondary o_domain_show_selection_button" type="button">
                    Show record(s)
            </button>
```